script/Deployer.sol

```diff
@@ -52,29 +52,29 @@ contract Deployer is Script {
         root = new Root{salt: salt}(address(escrow), delay, deployer);
 
         vaultFactory = address(new ERC7540VaultFactory(address(root)));
-        restrictionManagerFactory = address(new RestrictionManagerFactory(address(root)));
+        restrictionManager = address(new RestrictionManager{salt: salt}(address(root)));
```

Comment:
Deployed deterministically so Centrifuge Chain can store 1 restriction manager address as the default for all chains